packages/owlbot-bootstrapper/cli/cloudbuild-owlbot-bootstrapper.yaml

```diff
@@ -47,6 +47,8 @@ steps:
     - "$_SERVICE_CONFIG_PATH"
     - "--interContainerVarsPath"
     - "$_INTER_CONTAINER_VARS_PATH"
+    - "--test"
+    - "$_TEST"
```

Comment:
Arguments should be more descriptive rather than a state. Prefer `--skip-issue-on-failure` to a `--test` flag which could affect other unknown settings.